src/Workspaces/Core/Portable/FindSymbols/FindReferences/Finders/ConstructorInitializerSymbolReferenceFinder.cs

```diff
@@ -12,13 +13,14 @@
 
 namespace Microsoft.CodeAnalysis.FindSymbols.Finders
 {
-    internal class ConstructorInitializerSymbolReferenceFinder : AbstractReferenceFinder<IMethodSymbol>
+    internal sealed class ConstructorInitializerSymbolReferenceFinder : AbstractReferenceFinder<IMethodSymbol>
```

Comment:
i sealed a lot of stuff for my sanity.  it was harder to determine virtual/override routes otherwise.